Gu.Wpf.DataGrid2D/Source2D.Sources.cs

```diff
@@ -76,6 +76,35 @@ private static void OnColumnHeadersChanged(DependencyObject d, DependencyPropert
             {
                 return;
             }
+            
+            //Subscribe to Colection changed events
+              var action = new NotifyCollectionChangedEventHandler(
```

Comment:
Because the ` var dataGrid = (DataGrid)d;` is needed and the NotifyCollectionChanged event doesn't give the  datagrid it's bound to right? I think..